### PR TITLE
Date/Time Plugin

### PR DESCRIPTION
**Pull request by veranikaChyrun**

### CODE REVIEW

**Review by valfirst — COMMENTED** [review]
Please remove all noisy build system changes

File: docs/modules/plugins/partials/date-time-properties.adoc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+=== Required
+
+There are no required properties for the plugin.
+
+=== Optional
+
+`location.zone-id` - defines time zone which will be used. The default is set to a system default timezone. https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/ZoneId.html[Time zone format]
```
Comment (valfirst) [inline docs/modules/plugins/partials/date-time-properties.adoc:7]: I guess something went wrong with conflicts resolution, please, resolve all conflicts carefully

**Comment by ikalinin1** [comment]
@veranikaChyrun  please rebase

File: docs/modules/plugins/partials/date-time-diff-date-examples.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
+==== Examples of generation
```
Comment (ikalinin1) [inline docs/modules/plugins/partials/date-time-diff-date-examples.adoc:1]: can we put examples into 
/partials/examples/date-time/

**Comment by valfirst** [comment]
Please reopen once all comments are fixed and conflicts are resolved

**Pull request closed without merging**